lib/mix/tasks/phoenix.gen.channel.ex

```diff
@@ -52,6 +52,11 @@ defmodule Mix.Tasks.Phoenix.Gen.Channel do
     unless length(args) == 2 do
       raise_with_help
     end
+
+    args
```

Comment:
The generated module contains `Channel` at the end. So I wouldn't check it here but just after we call `Mix.Project.inflect`.